### PR TITLE
Create a data privacy document - please read

### LINKED ISSUE #1369: Add document detailing Data Privacy in design
Statement of what we have done/considered.
Write up of our review meeting

### PR DESCRIPTION
**Pull request by diversemix**
#### Background

Creates a document to detail data privacy

#### Any relevant tickets

Closes #1369

### Please approve to acknowledge you have read it - Thanks :smile:

### CODE REVIEW

**Review by hem-brahmbhatt — APPROVED** [review]

**Review by jyenes — APPROVED** [review]

File: docs/DATA_PRIVACY.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Please familiarise yourself with the following information:
+
+- [eLife's Privacy Notice](https://elifesciences.org/privacy)
+
+## Overview
+
+`elife-xpub` has been designed to integrate with [eLife's journal site](elifesciences.org).
+As such users control and manage the privacy of their data via their [ORCiD](orcid.org) account.
+eLife ensures that user data is kept up-to-date via the [Profiles Service](https://github.com/elifesciences/profiles)
+and its this service that `elife-xpub` interacts with to get all the user data.
```
Comment (giorgiosironi) [inline docs/DATA_PRIVACY.md]: `it's`?

File: docs/DATA_PRIVACY.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Review 2019-01-22
+
+Peter, Hannah reviewed the handling of data within `elife-xpub` as a precursor to launch.
```
Comment (giorgiosironi) [inline docs/DATA_PRIVACY.md]: this looks like a document with legal value (GDPR?), hence you may need surnames too?

**Review by giorgiosironi — APPROVED** [review]

File: docs/DATA_PRIVACY.md (hunk 1 above)
Comment (diversemix) [reply docs/DATA_PRIVACY.md]: :stuck_out_tongue_closed_eyes:  ok ok ok !

File: docs/DATA_PRIVACY.md (hunk 2 above)
Comment (diversemix) [reply docs/DATA_PRIVACY.md]: I think I will change this to "Product Manager" and "Technical Lead" ?

**Review by hdrury1 — APPROVED** [review]

**Review by SemraSh — APPROVED** [review]

**Pull request merged**